cookbooks/cron/recipes/default.rb

```diff
@@ -4,7 +4,7 @@
 #
 
 # Find all cron jobs specified in attributes/cron.rb where current node name matches instance_name
-crons = node[:custom_crons].find_all {|c| c[:instance_name].match "#{node[:name]}" }
+crons = node[:custom_crons].find_all {|c| c[:instance_name] =~ /#{node[:name]}/}
```

Comment:
The match is not anchored, so a name like 'redis1' would match redis1, redis10, redis11, etc...